function-invokers/java-function-invoker/src/main/java/io/sk8s/invoker/java/server/JavaFunctionInvokerController.java

```diff
@@ -31,8 +31,8 @@
 	@Autowired
 	private FunctionProperties functions;
 
-	@PostMapping("/")
-	public Object invoke() {
+	@RequestMapping("/")
+	public String invoke() {
```

Comment:
Why the change here? Ultimately, we want to support any kind of result, with HttpMessageConverter taking care of it

Reply:
Look at the method body. It only ever returns a String.

Reply:
Ah, sorry. Did not realize this was just the forward